orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -539,7 +541,8 @@ py::class_<TrainingAgent>(m, "TrainingAgent", R"pbdoc(This is the main class use
       .def_readwrite("initializer_grad_names_to_train", &GraphInfo::initializer_grad_names_to_train)
       .def_readwrite("user_output_names", &GraphInfo::user_output_names)
       .def_readwrite("output_grad_indices_non_differentiable", &GraphInfo::output_grad_indices_non_differentiable)
-      .def_readwrite("output_grad_indices_require_full_shape", &GraphInfo::output_grad_indices_require_full_shape);
+      .def_readwrite("output_grad_indices_require_full_shape", &GraphInfo::output_grad_indices_require_full_shape)
+      .def_readwrite("loss_gradient_names", &GraphInfo::loss_gradient_names);
```

Comment:
the output of module is not necessarily loss, it can be prediction, intermediate tensor, anything... 

A better name would be "module_output_gradient_names"